crates/iota-transaction-builder/src/lib.rs

```diff
@@ -631,6 +640,8 @@ impl TransactionBuilder {
         Ok(args)
     }
 
+    /// Build a TransactionKind::ProgrammableTransaction that contains
```

Comment:
put the enum variant in brackets as well?

Reply:
Done